content/blog/2023-10-05-october-Prism-launch/contents.lr

```diff
@@ -0,0 +1,22 @@
+title: Prism: The New Feinberg Repository for Global Dissemination of Research
+---
+pub_date: 2023-10-10
+---
+team: InvenioRDM
+---
+body:
+
+![](NEW_PICTURE_FILE_NEEDED.jpg)
+
+Feinberg School of Medicine has a newly available institutional repository called Prism. Prism preserves the intellectual works created by the Feinberg scholarly community and holds articles, conference presentations, preprints and other items created by faculty, staff, and students. Prism helps openness, maximizes reproducibility, and enhances research connections in the Feinberg community and globally to anyone with access to the internet. Prism will be available to Feinberg in April 2023.
+ 
+“We are excited to announce the launch of Prism as the institutional repository for Feinberg School of Medicine” says Karen Gutzman, Head of Research Assessment and Communications at Galter Health Sciences Library and Learning Center. Prism replaces the former DigitalHub and includes many much-anticipated features, such as the ability to create metadata-only records for offsite datasets, set embargo dates for releasing content to the public, creating and curating communities of practice, and sharing of private links to view and edit with colleagues. These new features complement existing features such as the ability to assign Digital Object Identifiers that make records citable, indexing by Google to make research widely discoverable, and a responsive staff at Galter Library to answer questions and provide support.
+
+Kristi Holmes, the Director of Galter Health Sciences Library and Learning Center and professor of Preventive Medicine, played a leading role in the development of Prism. According to Holmes, “It is essential to have a robust institutional repository that can keep up with the latest technologies and trends. As models for open access and data sharing continue to evolve, it's clear that institutional repositories will play an increasingly critical role to make research Findable, Accessible, Interoperable, and Reusable (FAIR).”
```

Comment:
Do we want to insert Kristi's "associate dean for knowledge management and strategy at Feinberg" title here? Could lend even more credence to the endeavor.